README.md

```diff
@@ -193,6 +193,7 @@ Slevomat Coding Standard for [PHP_CodeSniffer](https://github.com/squizlabs/PHP_
  - [SlevomatCodingStandard.Variables.UnusedVariable](doc/variables.md#slevomatcodingstandardvariablesunusedvariable)
  - [SlevomatCodingStandard.Variables.UselessVariable](doc/variables.md#slevomatcodingstandardvariablesuselessvariable-) 🔧
  - [SlevomatCodingStandard.Whitespaces.DuplicateSpaces](doc/whitespaces.md#slevomatcodingstandardwhitespacesduplicatespaces-) 🔧
+ - [SlevomatCodingStandard.Whitespaces.SpaceInNamedArgument](doc/whitespaces.md#slevomatcodingstandardspaceinnamedargument-) 🔧
```

Comment:
I think the `Functions` namespace would be better. There are already sniffs like `RequireTrailingCommaInCallSniff`